tests/property.test.ts

```diff
@@ -0,0 +1,117 @@
+import { describe } from 'vitest'
+import { test, fc } from '@fast-check/vitest'
+import age from '../lib/index.js'
+import { to_string } from 'libsodium-wrappers-sumo'
+
+fc.configureGlobal({
+  // increasing this value will make fast-check do more random test runs,
+  // which might be good for intensive testing, but also makes them slower.
+  numRuns: 50
+})
+
+const isEqualUInt8Array = (a: Uint8Array, b: Uint8Array) => {
+  if (a.byteLength !== a.byteLength) {
+    return false
+  }
+
+  for (let i = 0; i < a.byteLength; i++) {
+    if (a[i] !== b[i]) {
+      return false
+    }
+  }
+
+  return true
+}
+
+describe('Property Based Tests', () => {
+  describe('Asymmetric Encryption and Decryption', () => {
+    test.prop(
+      {
+        plaintext: fc.string()
+      }
+    )(
+      'decryption should invert encryption with identity/recipient (string plaintext)',
+      async ({ plaintext }) => {
+        const { Decrypter, Encrypter, generateIdentity, identityToRecipient } = await age()
+
+        const identity = generateIdentity()
+        const recipient = identityToRecipient(identity)
+
+        const enc = new Encrypter()
+        const dec = new Decrypter()
+
+        enc.addRecipient(recipient)
+        dec.addIdentity(identity)
+
+        return to_string(dec.decrypt(enc.encrypt(plaintext))) === plaintext
```

Comment:
Use `decrypt(..., "text")` instead of `to_string`.

Reply:
Changed :heavy_check_mark: